front-end/package.json

```diff
@@ -20,6 +20,8 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "cleanBuild": "rimraf ./build/*",
```

Comment:
Interesting, rimraf is like `rm -rf` right?

Reply:
Yep! Apparently it's node's version of a hard delete.